src/main/java/com/github/fge/jsonpatch/operation/OmitOperation.java

```diff
@@ -0,0 +1,27 @@
+package com.github.fge.jsonpatch.operation;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.fge.jackson.jsonpointer.JsonPointer;
+import com.github.fge.jsonpatch.operation.policy.PathMissingPolicy;
+
+/**
+ * Extended JSON Patch {@code omit} operation.
+ *
+ * The operation will remove the {@code value} at {@code path} if it exists.
+ * It will do nothing if the actual value at {@code path} is not equal to {@code value}.
+ * It will throw a "no such path" error if there is no value at {@code path}.
+ */
+public final class OmitOperation extends OmitOperationBase
+{
+    public static final String OPERATION_NAME = "omit";
+
+    @JsonCreator
+    public OmitOperation(@JsonProperty("path") final JsonPointer path,
+                         @JsonProperty("value") final JsonNode value)
+    {
+        super(OPERATION_NAME, path, value, PathMissingPolicy.THROW);
```

Comment:
I haven't really tried it, but I'm curious if like we talked about off line if we can turn operations into a *composed* combination of a source policy, destination policy, and an action.  So in this case you'd have an "OmitAction" and a "SourceRequired" being composed together.